src/components/button/Button.css

```diff
@@ -18,7 +18,31 @@
 .main-button:hover {
   background-color: #ffffff;
   color: black;
-  transition: ease-in 0.3s;
+  transition: ease-in 0.5s;
```

Comment:
I see you've increased the transition time, we don't need slower animations so let's revert this.

Reply:
Ok, thanks. Working on the outward animation
Also, I will revert the transition time.
Thanks